src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    /// <summary>
+    /// Tests of <see cref="TimeSyncBehavior"/> and <see cref="TimeSyncBehaviorState"/> classes.
+    /// </summary>
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        /// <summary>
+        /// Time data is added this this state. The combination of this and datetimeprovider are what is under test.
+        /// </summary>
+        private TimeSyncBehaviorState timesyncBehaviourState;
+
+        /// <summary>
+        /// This provides the adjusted time ready for assertion. The combination of this and timesyncBehaviorState are what is under test.
+        /// </summary>
+        private IDateTimeProvider dateTimeProvider;
```

Comment:
All those variables are defined in this file but used in the other one (`TimeSyncBehaviorAdjustedTimeTest_Steps.cs`)  so it's a bit hard to follow methods in there if the definition is done elsewhere. Can you please move those to a single file? 

I also don't think that having a partial class that is separated in 2 files is necessary.

Reply:
agree - merged the two files back into one